### DISCUSSION TITLE
config.h file not availabe for AIX 7.1

### DISCUSSION THREAD

**Discussion by ramueedupalli** [DOCGroup/ACE_TAO#2159] [Q&A] [answered]
Hi,

I am trying to build ACE and TAO on AIX 7.1 with xlC_r (IBM XL C/C++ for AIX, V12.1 ) compiler. But I don't find relevant config.h file for aix platform. Can someone help me with this.

**Comment by jwillemsen** [comment, marked as answer]
AIX support on master has been removed due to lack of maintenance (see  issue https://github.com/DOCGroup/ACE_TAO/issues/2018 and pull request https://github.com/DOCGroup/ACE_TAO/pull/2031). You could try the latest ACE 6.5.x release, see https://github.com/DOCGroup/ACE_TAO/releases but keep in mind that AIX hasn't been maintained and tested for years.

**Reply by ramueedupalli** [reply]
Thank you!
